src/app/submission/shared/submission.model.ts

```diff
@@ -614,8 +618,22 @@ export class Sections extends HasUpdates<UpdateEvent> {
         return s;
     }
 
-    remove(): void {
-        // TODO
+    remove(section: Section): void {
```

Comment:
Is that still necessary? Since I opted to perform the check on the digest cycle through ngIf, by the time a subsection is rendered it will be guaranteed to only have the delete button if the condition is met (a.k.a is not required or is not the last one). Therefore, the "remove" method is only reached if the removal is allowed. At least that's my understanding.